examples/burial-form/ReviewPage.jsx

```diff
@@ -349,7 +350,7 @@ export default function ReviewPage(props) {
   }
 
   return (
-    <>
+    <Page {...props} nextPage="/confirmation">
```

Comment:
Do you need to put a previous page path here?

Reply:
Yep, I should probably add it here